### PR TITLE
Line-properties

### PR DESCRIPTION
**Pull request by YavorLalev**
The user can customise the line type in the graph. The following properties can be changed: line color, line style and line width.

[US-Line properties](https://github.com/users/catdieval/projects/1/views/1?pane=issue&itemId=55809973)

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -115,6 +120,18 @@ export default function App({ Component, pageProps }) {
...
+  function handleLineStyleChange(event) {
+    setLineStyle(event.target.value);
+  }
+
+  function handleLineWidthChange(event) {
+    setLineWidth(event.target.value);
+  }
+
   return (
     <>
```
Comment (onemanwenttomow) [inline pages/_app.js:136]: No need for the React Fragment here, as you are only returning 1 element (`Layout`)

File: pages/_app.js [hunk 2]
```diff
@@ -38,6 +38,11 @@ export default function App({ Component, pageProps }) {
   //Step 5: Add a title to the graph
   const [titleLabel, setTitleLabel] = useState("");
 
+  //Line Properties
+  const [lineColor, setLineColor] = useState("");
+  const [lineStyle, setLineStyle] = useState("");
+  const [lineWidth, setLineWidth] = useState(0);
```
Comment (onemanwenttomow) [inline pages/_app.js:44]: It might be that you want this... but i need to check. Do you want the default value to be 0?

**Review by onemanwenttomow — APPROVED** [review]
Nice job team 🎉

Added a couple of very minor comments, but consider this approved.

File: Components/DropDownMenu/index.js [hunk 3]
```diff
@@ -1,15 +1,25 @@
...
+import { Container } from "../Container";
+
+const StyledLabel = styled.label`
+  font-weight: 400;
+  margin: 0.3rem;
+`;
+
 export default function DropDownMenu({ idString, onChange, arrayOfOptions }) {
   return (
     <>
```
[the remark this answers is not in the document]
Comment (YavorLalev) [reply Components/DropDownMenu/index.js:11]: Done.

File: pages/_app.js (hunk 1 above)
Comment (YavorLalev) [reply pages/_app.js]: Done.

**Pull request merged**